internal/cli/status.go

```diff
@@ -0,0 +1,992 @@
+
+	workspace, err := c.getWorkspaceFromProject(projResp)
+	if err != nil {
+		return err
+	}
+
+	// App Summary
+	//  Summary of single app
+	var app *pb.Application
+	for _, a := range project.Applications {
+		if a.Name == appTarget {
+			app = a
+			break
+		}
+	}
+	if app == nil {
+		return fmt.Errorf("Did not find application %q in project %q", appTarget, projectTarget)
+	}
+
+	// Deployment Summary
+	//   Deployment List
+
+	respDeployList, err := client.ListDeployments(c.Ctx, &pb.ListDeploymentsRequest{
+		Application: &pb.Ref_Application{
+			Application: app.Name,
+			Project:     project.Name,
+		},
+		Workspace: &pb.Ref_Workspace{
+			Workspace: workspace,
+		},
+		LoadDetails: pb.Deployment_BUILD,
+	})
+	if err != nil {
+		return err
+	}
+
+	// Pregrab status report list
+	statusReportsResp, err := client.ListStatusReports(c.Ctx, &pb.ListStatusReportsRequest{
+		Application: &pb.Ref_Application{
+			Application: app.Name,
+			Project:     project.Name,
+		},
+		Workspace: &pb.Ref_Workspace{
+			Workspace: workspace,
+		},
+	})
+	if status.Code(err) == codes.NotFound || status.Code(err) == codes.Unimplemented {
+		err = nil
+		statusReportsResp = nil
+	}
+	if err != nil {
+		return err
+	}
+
+	deployHeaders := []string{
+		"App Name", "Version", "Workspace", "Platform", "Details", "Lifecycle State",
+	}
+
+	deployTbl := terminal.NewTable(deployHeaders...)
+
+	resourcesHeaders := []string{
+		"Type", "Platform", "Category",
+	}
+
+	resourcesTbl := terminal.NewTable(resourcesHeaders...)
+
+	deployStatusReportComplete := "N/A"
+	var deployStatusReportCheckTime string
+	appFailures := false
+	if len(respDeployList.Deployments) > 0 {
+		deploy := respDeployList.Deployments[0]
+		statusColor := ""
+
+		var details string
+		if img, ok := deploy.Preload.Build.Labels["common/image-id"]; ok {
+			img = shortImg(img)
+
+			details = "image:" + img
+		}
+
+		columns := []string{
+			deploy.Application.Application,
+			fmt.Sprintf("v%d", deploy.Sequence),
+			deploy.Workspace.Workspace,
+			deploy.Component.Name,
+			details,
+			deploy.Status.State.String(),
+		}
+
+		// Add column data to table
+		deployTbl.Rich(
+			columns,
+			[]string{
+				statusColor,
+			},
+		)
+
+		// get deploy status report
+		statusReport, err := c.getLatestStatusReportByDeployId(statusReportsResp, deploy.Id)
+		if status.Code(err) == codes.NotFound {
+			err = nil
+			statusReport = nil
+		}
+		if err != nil {
+			return err
+		}
+		deployStatusReportComplete, deployStatusReportCheckTime, err = c.FormatStatusReportComplete(statusReport)
+		if err != nil {
+			return err
+		}
+
+		// Deployment Resources Summary
+		//   Resources List
+		// TODO(briancain): Add resource health when it exists
+		if statusReport != nil {
+			if statusReport.Health.HealthStatus == "ERROR" ||
+				statusReport.Health.HealthStatus == "DOWN" {
+				appFailures = true
+			}
+
+			for _, dr := range statusReport.Resources {
+				columns := []string{
+					dr.Name,
```

Comment:
You're putting `Name` in the `Type` column here.

Reply:
Fixed!